docs/development/local_setup.md

```diff
@@ -1,67 +1,81 @@
+The Gardener API server extends the Kubernetes API via the user-aggregated API server concepts.
 However, if you want to develop it, you may want to work locally with the Gardener without building a Docker image and deploying it to a cluster each and every time.
 That means that the Gardener runs outside a Kubernetes cluster which requires providing a [Kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/) in your local filesystem and point the Gardener to it when starting it (see below).
 
-Further details could be found in
+Further details can be found in
 
 1. [Principles of Kubernetes](https://kubernetes.io/docs/concepts/), and its [components](https://kubernetes.io/docs/concepts/overview/components/)
 1. [Kubernetes Development Guide](https://github.com/kubernetes/community/tree/master/contributors/devel)
 1. [Architecture of Gardener](https://github.com/gardener/documentation/wiki/Architecture)
 
-## Installing Golang environment
+This guide is split into three main parts:
+* Preparing your setup by installing all dependencies and tools
+* Building and starting Gardener components locally
+* Using your local Gardener setup to create a Shoot
 
-Install latest version of Golang. For MacOS you could use [Homebrew](https://brew.sh/):
+## Limitations of the local development setup
 
-```bash
-brew install go
-```
+You can run Gardener (API server, controller manager, scheduler, gardenlet) against any local Kubernetes cluster, however, your seed and shoot clusters must be deployed to a cloud provider.
+Currently, it is not possible to run Gardener entirely isolated from any cloud provider. This means that to be able create Shoot clusters you need to register an external Seed cluster (e.g., one created in AWS).
 
-For other OS, please check [Go installation documentation](https://golang.org/doc/install).
+# Preparing the Setup
 
-## Installing kubectl and helm
+## [macOS only] Installing homebrew
 
-As already mentioned in the introduction, the communication with the Gardener happens via the Kubernetes (Garden) cluster it is targeting. To interact with that cluster, you need to install `kubectl`. Please make sure that the version of `kubectl` is at least `v1.11.x`.
+The copy-paste instructions in this guide are designed for macOS and use the package manager [Homebrew](https://brew.sh/).
 
-On MacOS run
+On macOS run
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+## Installing git
+
+We use `git` as VCS which you need to install. On macOS run
 
 ```bash
-brew install kubernetes-cli
+brew install git
 ```
 
-Please check the [kubectl installation documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for other OS.
+On other OS, please check the [Git installation documentation](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
```

Comment:
```suggestion
For other OS, please check the [Git installation documentation](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
```

You can the same msg everywhere. Now you have `For other OS, please check`, `On other OS please check`, `Please check [...] for other OS`.

Reply:
done